internal/server/authn/method/oidc/server.go

```diff
@@ -223,10 +223,15 @@ func (s *Server) providerFor(provider string, state string) (*capoidc.Provider,
 		return nil, nil, err
 	}
 
+	nonce := providerCfg.Nonce
+	if nonce == "" {
+		nonce = "static"
+	}
+
```

Comment:
Could you please move this to `AuthenticationMethodOIDCProvider.setDefaults`. Please use `AuthenticationMethodKubernetesConfig` as a reference. https://github.com/flipt-io/flipt/blob/dd6a0a0924c7cced7cdbd3cb644e23f9359daf31/internal/config/authentication.go#L538-L542

Reply:
Of course!

Reply:
@GeorgeMac Could you please advice on this matter. We move it to config and set defaults but OIDC is a unique here as providers are custom keys and I really don't see easy way to set `nonce` to `static` right now (my bad). Maybe you have some idea.